assets/canary/daemonset.yaml

```diff
@@ -10,6 +10,7 @@ spec:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
     spec:
+      priorityClassName: openshift-cluster-critical
```

Comment:
Should that be `system-cluster-critical`?  Where is "`openshift-cluster-critical`" defined?

Reply:
Yes! Typo :sweat: